examples/platforms/raspbian/default

```diff
@@ -33,3 +33,4 @@ DNS64=1
 DHCPV6_PD=1
 NETWORK_MANAGER=1
 BACKBONE_ROUTER=0
+ADVERTISING_PROXY=0
```

Comment:
We may want to change those default values in the future to enable DUckhorn Border Router as default, @librasungirl

Reply:
removed this for now as it is not necessary for the Advertising Proxy.